ios/FluentUI.Demo/FluentUI.Demo/Demos/PeoplePickerDemoController.swift

```diff
@@ -39,6 +39,7 @@ class PeoplePickerSampleData {
 
 class PeoplePickerDemoController: DemoController {
     var peoplePickers: [PeoplePicker] = []
+    var dividers: [MSFDivider] = []
```

Comment:
Does this need to be an array if it only has one value? Or is there a chance that there will be more in the future?

Reply:
There's already multiple. There's 4 variations of the PeoplePickerSampleData, so we add three dividers during viewDidLoad.